schedule/schedule-appengine/src/main/client/jasify/admin/admin.routes.js

```diff
@@ -381,14 +381,7 @@
 
                         function allowed() {
                             if ($route.current.params.organizationId) {
-
-                                var dfd = $q.defer();
-
-                                Multipass.query($route.current.params.organizationId).then(function (result) {
-                                    dfd.resolve(angular.extend({items: []}, result));
-                                });
-
-                                return dfd.promise;
```

Comment:
All endpoint methods already return a promise.  The implementation above works perfectly fine, but I find the following more simple:
~~~javascript
return Multipass.query(..)
    .then(function (result) {
         return angular.extend(....);
     });
~~~

Or, if you want to continue returning a `then` able promise, you can return `$q.when(...)` like so:

~~~javascript
return Multipass.query(..)
    .then(function (result) {
         return $q.when(angular.extend(....));
     });
~~~


I will try the first approach